cowprotocol/cowswap
# Feature/limit UI update -  #2166 hide unlikely to execute

# Summary

Closes #2166 

Hide high fee warning when estimated execution price < spot price

# To Test

1. Place an order with a small amount and a bad price for you (below market)
2. Wait for the fee to be > 10% of your sell amount
* Assuming backend returns a price, you should see the green indicator, no fee warning indicator and the tooltip should say "will execute soon"

![image](https://user-images.githubusercontent.com/43217/227600458-fee451b3-9d40-4ae7-83bf-fcebcad27af2.png)
